test/functional/paging.ts

```diff
@@ -80,35 +80,35 @@ suite<Paging>('gb-paging', ({ html, mount }) => {
     expect((<HTMLImageElement>html().querySelector('.gb-pager__link.prev img')).src).to.contain(prev_icon);
   });
 
-  describe('allowed paging behvaiour', () => {
+  describe('allowed paging behaviour', () => {
     it('should be able to page backward', () => {
       const tag = mount();
 
-      tag.updatePages({ pageIndex: 2, finalPage: 4 });
+      tag.update(tag.pageInfo([1, 2, 3, 4, 5], 2, 5));
```

Comment:
use an object for test instead of `tag.pageInfo()` so that changes to that function don't affect your tests